src/models/recipient.ts

```diff
@@ -0,0 +1,6 @@
+export interface Recipient {
+    stopID: string;
```

Comment:
In our documentation we write id in camelCase `stopId`, should we change the syntax?

Reply:
damn